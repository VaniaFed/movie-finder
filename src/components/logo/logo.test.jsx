import renderer from 'react-test-renderer';
import { Logo } from './index';

describe('<Logo />', () => {
    const tree = renderer.create(<Logo text="this is some logo" />).toJSON();
    it('should be rendered with text "this is some logo"', () => {
        expect(tree).toMatchSnapshot();
    });
});
