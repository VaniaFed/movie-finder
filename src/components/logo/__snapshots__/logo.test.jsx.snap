// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should be rendered with text "this is some logo" 1`] = `
<h1
  className="logo"
>
  this is some logo
</h1>
`;
